native/src/main/java/io/ballerina/stdlib/sql/Constants.java

```diff
@@ -181,6 +181,8 @@ private SqlTypes() {}
         public static final String DECIMAL_TYPE = "decimal";
         public static final String OPTIONAL_BYTE = "byte[]?";
         public static final String BYTE_ARRAY_TYPE = "byte[]";
+        public static final String CIVIL_ARRAY_TYPE = "time:Civil";
+        public static final String TIME_OF_DAY_ARRAY = "time:TimeOfDay";
```

Comment:
Should this be `TIME_OF_DAY_ARRAY_TYPE`?

Reply:
Changed to TIME_OF_DAY_ARRAY_TYPE.